app/experimenter/nimbus-ui/src/components/DirectoryTable/index.tsx

```diff
@@ -0,0 +1,169 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { getAllExperiments_experiments } from "../../types/getAllExperiments";
+import "./index.scss";
+import { Link } from "@reach/router";
+import LinkExternal from "../LinkExternal";
+import {
+  getProposedEndDate,
+  getProposedEnrollmentRange,
+  humanDate,
+} from "../../lib/dateUtils";
+
+// These are all render functions for column type sin the table.
+export type ColumnComponent = React.FC<getAllExperiments_experiments>;
+
+export const DirectoryColumnTitle: ColumnComponent = ({ slug, name }) => {
+  return (
+    <td className="w-33">
+      <Link to={slug} data-sb-kind="pages/Summary">
+        {name}
+      </Link>
+      <br />
+      <span className="text-monospace text-secondary small">{slug}</span>
+    </td>
+  );
+};
+
+export const DirectoryColumnOwner: ColumnComponent = ({ owner }) => (
+  <td>{owner && owner.username}</td>
+);
+
+export const DirectoryColumnFeature: ColumnComponent = ({ featureConfig }) => (
+  <td>
+    {featureConfig ? (
+      <>
+        {featureConfig.name}
+        <br />
+        <span className="text-monospace text-secondary small">
+          {featureConfig.slug}
```

Comment:
Q: Do we want to display the `.slug` or `.name` here?

Reply:
Judging by the styling I think slug is intentional, and .name is just up abive

Reply:
both, since some folks  need the slug for DS reasons and whatnot